scripts/install_third_party_libs.py

```diff
@@ -88,8 +88,12 @@
     common.OPPIA_TOOLS_DIR, 'buf-%s' % common.BUF_VERSION)
 PROTOC_DIR = os.path.join(BUF_DIR, 'protoc')
 # Path of files which needs to be compiled by protobuf.
+# Paired with the frontend requirement.
```

Comment:
Don't know what this comment means; please elaborate. What is the "frontend requirement"?

Reply:
Added explanation.

```
# List of a tuple represents the path of files that needs to be compiled by protobuf
# and a boolean value which represents whether buf needs to generate
# its js and ts file.
```